app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  after_save :update_post_counter
+
+  private
+
+  def update_post_counter
```

Comment:
[OPTIONAL] I think you should rename this method to update_likes_counter to avoid confusion